test/e2e/tags.go

```diff
@@ -0,0 +1,15 @@
+package e2e
+
+// The use of ginkgo tags is encouraged for describing tests.
+// This file defines test tags and annotations.
+// Each annotation should have a corresponding description.
+const (
+
+	// May fail in secured clusters which aren't specifically opened up to allow certain operations.
+	InsecureTestTag = "[NonStandardSecurity]"
+
+	// Required for any cluster to be defined as a functional Kubernetes cluster.  This is a WIP.
+	ConformanceTestTag = "[Conformance]"
```

Comment:
Going to have to defer to others whether it's a good idea to extract these constants.  Generally I would say yes, but I'm not a fan of pulling it out unless it's worth doing across the codebase.  Maybe out of scope for this PR?